cql3/expr/expression.cc

```diff
@@ -1707,6 +1707,33 @@ cql3::raw_value evaluate(const field_selection& field_select, const evaluation_i
     return field_value;
 }
 
+static
+cql3::raw_value
+evaluate(const column_mutation_attribute& cma, const evaluation_inputs& inputs) {
+    auto col = expr::as_if<column_value>(&cma.column);
+    if (!col) {
+        on_internal_error(expr_logger, fmt::format("evaluating column_mutation_attribute of non-column {}", cma.column));
+    }
+    int32_t index = inputs.selection->index_of(*col->col);
```

Comment:
It would be nice to check that this column exists, otherwise we will get a crash when it doesn't.

Reply:
prepare should create self-consistent data.

After we clean up the code some more, prepare can store the index directly in the column_value thing.